front/app/api/custom_fields/util.ts

```diff
@@ -30,7 +30,6 @@ const properties = [
   'linear_scale_label_9_multiloc',
   'linear_scale_label_10_multiloc',
   'linear_scale_label_11_multiloc',
-  'question_category',
   'maximum',
```

Comment:
This was causing a bug in the form builder, so I've removed it from this list as it's not always present on a custom field.